payload-examples/api.github.com/installation_repositories.added.payload.without-installation-prop.json

```diff
@@ -0,0 +1,34 @@
+{
+  "action": "added",
+  "repository_selection": "selected",
+  "repositories_added": [
+    {
+      "id": 186853007,
+      "node_id": "MDEwOlJlcG9zaXRvcnkxODY4NTMwMDc=",
+      "name": "Space",
+      "full_name": "Codertocat/Space",
+      "private": false
+    }
+  ],
+  "repositories_removed": [],
+  "sender": {
+    "login": "Codertocat",
+    "id": 21031067,
+    "node_id": "MDQ6VXNlcjIxMDMxMDY3",
+    "avatar_url": "https://avatars1.githubusercontent.com/u/21031067?v=4",
+    "gravatar_id": "",
+    "url": "https://api.github.com/users/Codertocat",
+    "html_url": "https://github.com/Codertocat",
+    "followers_url": "https://api.github.com/users/Codertocat/followers",
+    "following_url": "https://api.github.com/users/Codertocat/following{/other_user}",
+    "gists_url": "https://api.github.com/users/Codertocat/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/Codertocat/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/Codertocat/subscriptions",
+    "organizations_url": "https://api.github.com/users/Codertocat/orgs",
+    "repos_url": "https://api.github.com/users/Codertocat/repos",
+    "events_url": "https://api.github.com/users/Codertocat/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/Codertocat/received_events",
+    "type": "User",
+    "site_admin": false
+  }
+}
```

Comment:
The `installation_repositories` event has always the `installation` key.

Reply:
I doubted about this one because, according to docs, it says
> The GitHub App installation. Webhook payloads contain the installation property when the event is configured for and sent to a GitHub App.

https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads#installation_repositories

Reply:
go PR the docs, I'm very sure that `"installation"` is always present for these two particular events. They will let us know if otherwise. You can reference this PR